test/scripts/initialize_eks_cluster.bash

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-EKS_CLUSTER_NAME=$1
-IMAGE_VERSION=$2
+AWS_REGION=$1
```

Comment:
Same here too. Maybe declare a variable here with the default value being `us-west-2` with an option to override by the caller?